exhale/utils.py

```diff
@@ -834,3 +834,74 @@ def fancyError(critical_msg=None, lex="py3tb", singleton_hook=None):
             ))
 
     os._exit(1)
+
+
+def groupsFromBalancedBrackets(string : str, left_bracket : str, right_bracket : str):
```

Comment:
Scary function, but it basically takes a string, and turns it into a list of characters, except when the character is a bracket, it'll add a list.
So:
```py
groups = groupsFromBalancedBrackets('Image< 1920, 1080', '<', '>')
```
becomes
```
['I', 'm', 'a', 'g', 'e', [' ', '1', '9', '2', '0', ',', ' ', '1', '0', '8', '0', ' ']]
```
This is later used to create the template groups correctly.

Reply:
If you want to have a play around with it, you can try it online here:
https://godbolt.org/z/cG4zEM3Y4